src/main/java/org/liquidengine/legui/component/Widget.java

```diff
@@ -684,7 +684,8 @@ public boolean isResizable() {
     }
 
     public void setResizable(boolean resizable) {
-        this.resizeButton.setVisible(this.resizable = resizable);
+        this.resizable = resizable;
+        this.resizeButton.getStyle().setDisplay(resizable ? DisplayType.MANUAL : DisplayType.NONE);
```

Comment:
Should be display `FLEX` in case of true.

Reply:
why the resize button needs to be FLEX?

Reply:
Sorry, my mistake